app/views/layouts/application.html.erb

```diff
@@ -29,8 +29,14 @@
   <body class="flex flex-col min-h-screen h-full bg-gray-100">
     <div class="flex-1 bg-gray-100">
       <%= render partial: 'layouts/nav' %>
-
-      <div class="py-10">
+      <div class="pt-2">
```

Comment:
You should move all of these wrappers inside the partial. Otherwise, they will space out the content when there is no announcement.